app/settings/__init__.py

```diff
@@ -13,6 +13,7 @@ class Settings(BaseModel):
     listener_location: ListenerLocation
     notification_thresholds: NotificationThresholds
     udp_broadcast: UDPBroadcast
+    fetch_data: bool = Field(default=True)
```

Comment:
the name of this variable is not very clear, it should be changed to something else. like “fetch_from_online”